assets/custom.js

```diff
@@ -113,6 +113,70 @@ $('.product-form').on('submit', function(e) {
     })
 })
 
+
+//Number to currency formatter
+const currency = new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+});
+
+// Adds or Subtracts 1 from Item Quantity on Cart page when user clicks on '+' or '-'. Minimum quantity is '0'
+$(".item__quantity p").click(function(e){
+    let itemQuantityInput = $(this).closest('.quantity-input').find('.itemQuantityInput');
+    let variantId = $(this).closest('.quantity-container').find('.variant-id').val();
+    let totalPrice = $(this).closest('.line-item').find('.final-price');
+    let currentQuantityInt = parseInt(itemQuantityInput.val());
+    if ($(e.target).hasClass('add')) {
```

Comment:
when using jquery, you can just use `$(this)` instead of using `$(e.target)` --> `$(this).hasClass('add')`